var/spack/repos/builtin/packages/edm4hep/package.py

```diff
@@ -93,6 +93,14 @@ class Edm4hep(CMakePackage):
 
     conflicts("%clang@:16", when="@0.99.1:", msg="Incomplete consteval support in clang")
 
+    # Fix missing nljson import
+    # NOTE that downstream packages (dd4hep) may fail for 0.99 and before
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/key4hep/EDM4hep/pull/379.patch?full_index=1",
+        when="@0.99.1",
```

Comment:
This might also apply for `0.99`, but I haven't tested it. In that case it would at least make the behavior the same for the same minor version (re comment about dd4hep here).

Reply:
I did test it for .99 since I had defaulted to cxx17, and the patch failed. It might not be hard to make a new patch for it but I figured anyone who wants to use edm4hep is probably using the latest version anyway since it's rapidly developing.

Reply:
Thanks for the clarification. I am definitely all for pushing people to use the latest version ;) I suppose if anyone ever needs compatibility with 0.99, we can add the patch then.